solutions/swb-app/src/audit/swbAuditLogger.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuditEntry, Writer } from '@aws/workbench-core-audit';
+
+export default class SwbAuditLogger implements Writer {
+  public async write(metadata: unknown, auditEntry: AuditEntry): Promise<void> {
+    const updatedAuditEntry = { ...auditEntry };
+    delete updatedAuditEntry.statusCode;
+    console.log(updatedAuditEntry);
```

Comment:
By default, this will log to CWL. `console.log` will convert the object to a JSON object. The output looks like this
```
2022-12-14T15:37:50.489Z	8247f646-3048-4a9d-9499-203d33a62cd3	INFO	{
  timestamp: 1671032270489,
  awsRequestId: '8247f646-3048-4a9d-9499-203d33a62cd3',
  logEventType: 'audit',
  statusCode: 201,
  action: 'POST /costCenters',
  actor: { uid: '1dc5a73f-efac-4abc-85c6-9826838ab510' },
  source: { ip: '72.83.25.116, 130.176.179.84' },
  body: {
    name: 'CostCenter-3',
    accountId: 'acc-2272f4a8-1791-419a-8e49-b926f3eb46f9',
    description: 'Description for CostCenter-3'
  }
}
```